index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="CSS/Styles.css">
+    <title>My Personal Portfolio Project</title>
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <header>
+        <div class="header">
+            <img class="logo" src="Images/image_geometry_1.png" alt="Top Geometric Figures">
+            <a href="#"><img class="menu" src="Images/ic_menu.png" alt="mobile-menu"></a>
+        </div>
+    </header>
+    <main>
+        <div class="headline">
+            <h1>Charlie Giorgania</h1>
+            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+            <button class="collab">Start collaboration</button>
+            <img class="deco" src="Images/image_geometry_2.png" alt="Colorful Geometric Figures">
```

Comment:
The comment for the above `<img>` tag applies here as well.

<img width="691" alt="Screen Shot 2022-05-25 at 17 25 09" src="https://user-images.githubusercontent.com/82426682/170379185-04e12acb-3519-431d-9586-9d1500ac5d27.png">